project/Dependencies.scala

```diff
@@ -119,12 +118,15 @@ object Dependencies {
     * and manually uploaded to the Broad Institute artifactory at https://broadinstitute.jfrog.io/broadinstitute/.
     * Consider updating to the official newer Slick version once they fix issue #2076
     * Related Slick PR: https://github.com/slick/slick/pull/2101
+    *
+    * Update 2022-03-23: This #2201 PR cherry picks Greg's #2101 PR above and claims to fix the issue:
+    * https://github.com/slick/slick/pull/2201
   */
-  private val slickV = "3.3.2-2076hotfix" // scala-steward:off (CROM-6620)
+  private val slickV = "3.4.0-M1"
```

Comment:
I am fairly sure this `3.4.0-M1` incorporates Greg's fixes but that should be confirmed.

Reply:
I agree with your comment that the fixes from [#2101 ](https://github.com/slick/slick/pull/2101) are included in `3.4.0-M1`, and match the changes on [Greg's branch](https://github.com/grsterin/slick/commits/v3.3.2-2076hotfix). Do you think it's worth confirming more than that before releasing this? Can we easily check "whether the new Slick version passes tests with upserts enabled (eg KeyValueDatabaseSpec)"?

Reply:
Yes `KeyValueDatabaseSpec` does run successfully against the MySQL containers on my local machine (it doesn't look like we're running tests tagged `DbmsTest` in the `services` sub-build in Travis).